src/jira.js

```diff
@@ -861,6 +861,26 @@ export default class JiraApi {
     }));
   }
 
+  /** Update comment for an issue
+   * [Jira Doc](https://docs.atlassian.com/jira/REST/cloud/#api/2/issue-updateComment)
+   * @name updateComment
+   * @function
+   * @param {string} issueId - Issue with the comment
+   * @param {string} commentId - Comment that is updated
+   * @param {string} comment - string containing new comment
+   */
+  updateComment(issueId, commentId, comment) {
+    return this.doRequest(this.makeRequestHeader(this.makeUri({
+      pathname: `/issue/${issueId}/comment/${commentId}`,
+    }), {
+      body: {
+        body: comment,
```

Comment:
There are other options that can be specified in this request other than just the comment, my suggestion is to add a fourth argument to this method, call it options, and add it in here using the spread (`...`) operator.  Other than that, this PR looks good.  😄